docs/sources/companion-plugins/companion-options.mdx

```diff
@@ -0,0 +1,69 @@
+---
+slug: /companion-options
```

Comment:
Can you turn this into an MDX component and not a separate page? Then you can just inline the component in all doc pages

Reply:
done!